core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceUnitTests.scala

```diff
@@ -1573,7 +1573,7 @@ class WorkspaceServiceUnitTests
   behavior of "updateAcl"
   it should "call Sam for Rawls workspaces" in {
     val projectOwnerEmail = "projectOwner@example.com"
-    val ownerEmail = "owner@example.com"
+    val ownerEmail = ctx.userInfo.userEmail.value // user making the request is an owner
```

Comment:
In all of these test cases, the user calling `updateACL` didn't have access to the workspace.